objgraph.py

```diff
@@ -582,9 +582,14 @@ def at_addrs(address_set):
     .. versionadded:: 3.4
     """
     res = []
-    for o in gc.get_objects():
-        if id(o) in address_set:
-            res.append(o)
+    id_to_obj = dict((id(o), o) for o in gc.get_objects())
```

Comment:
I suggest

    id_to_obj = {id(o): o for o in gc.get_objects()}

Reply:
:+1: